bootstrap.sh

```diff
@@ -48,6 +48,24 @@ mkdir -p $VTROOT/vthook
 echo "Updating git submodules..."
 git submodule update --init
 
+# Install "protoc" protobuf compiler binary.
+protoc_version=3.4.0
+protoc_dist=$VTROOT/dist/protoc
+protoc_version_file=$protoc_dist/version
+if [[ -f $protoc_version_file && "$(cat $protoc_version_file)" == "$protoc_version" ]]; then
+  echo "skipping protoc install. remove $protoc_version_file to force re-install."
+else
+  rm -rf $protoc_dist
+  mkdir -p $protoc_dist
+  download_url=https://github.com/google/protobuf/releases/download/v${protoc_version}/protoc-${protoc_version}-linux-x86_64.zip
```

Comment:
We'll need to handle mac right?

Reply:
Yes.

I'll do this in a separate PR. Tracking it as part of: https://github.com/youtube/vitess/issues/3229#issuecomment-331550432